extensions/amp-ima-video/amp-ima-video.md

```diff
@@ -0,0 +1,74 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-ima-video"></a> `amp-ima-video`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A video player with an integration of the
+    <a href="https://developers.google.com/interactive-media-ads/docs/sdks/html5/">IMA SDK</a>
+    for instream video ads.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Experiment</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-ima-video" src="https://cdn.ampproject.org/v0/amp-ima-video-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>fixed, responsive</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td>None</td>
+  </tr>
+</table>
+
+## Overview 
+
+You can use the `amp-ima-video` component to embed an IMA SDK enabled video
```

Comment:
- Add link to the [IMA SDK](https://developers.google.com/interactive-media-ads/docs/sdks/html5/) 
- Rewrite the second sentence (and make it as a paragraph) with these minor tweaks:

To embed a video, provide a source URL for your content video (`data-src`) and an ad tag (`data-tag`), which is a URL to a VAST-compliant ad response (for examples, see [IMA Sample Tags](https://developers.google.com/interactive-media-ads/docs/sdks/html5/tags)).